info/v1/container.go

```diff
@@ -41,6 +41,10 @@ type MemorySpec struct {
 	SwapLimit uint64 `json:"swap_limit,omitempty"`
 }
 
+type PidsSpec struct {
```

Comment:
Can we call this ProcessSpec for consistency?

Reply:
ok, I will move it